arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json

```diff
@@ -259,13 +259,13 @@
         ],
         "responses": {
           "200": {
-            "description": "the operation completed successfully.",
+            "description": "The operation completed successfully.",
             "schema": {
               "$ref": "#/definitions/GatewayResource"
             }
           },
           "default": {
-            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema, and an exception will be thrown.",
```

Comment:
Typo in "defintion". I'm not going to do any more comments for these, so you might want to run spell check on the entire file.